source/agora/test/Script.d

```diff
@@ -65,21 +66,17 @@ unittest
     lock_txs.each!(tx => nodes.each!(node => node.putTransaction(tx)));
     network.expectHeightAndPreImg(Height(6), network.blocks[0].header);
 
-    Unlock keyUnlocker (in Transaction tx, in OutputRef out_ref) @safe nothrow
-    {
-        auto pair = SigPair(WK.Keys.Genesis.sign(tx.getChallenge()), SigHash.All);
-        return Unlock([ubyte(65)] ~ pair[].dup);
-    }
```

Comment:
This was original Genesis key `keyUnlocker` code that I forgot to originally remove in this commit.

Reply:
@omerfirmak why was the first `ubyte` 65 here?

Reply:
It was `SigPair.sizeof`, essentially a `PUSH` opcode for 65 bytes.
but serilalized SigPair is not constant size anymore, `toPushOpcode(pair[])` is the right way to handle that.